src/ui/sass/modules/_FilterOptions.scss

```diff
@@ -217,6 +221,10 @@
     margin-left: 0;
   }
 
+  &-radioButtonInput {
+    cursor: pointer;
+  }
+
```

Comment:
Would it make sense for this styling to go on the parent of `&-radioButtonInput` and `&-radioButtonLabel`? Thinking that maybe later we'll add margin between these elements and if the area between them is clickable then it may need a cursor: pointer too.

Reply:
just tried adding that, but it seemed a little weird because clicking that array wouldn't actually toggle the option